### PR TITLE
feat: Add Oracle dialect specific documentation V7

### PR DESCRIPTION
**Pull request by sudarshan12s**
* feat: Add Oracle dialect specific documentationOracle Docs for V7

### CODE REVIEW

File: docs/other-topics/other-data-types.mdx [hunk 1]
```diff
@@ -27,61 +27,61 @@ A ❌ means the dialect does not support that DataType.
...
-| `TSVECTOR`                                                                         | [`TSVECTOR`][postgres-tsvector]                                 | ❌                                  | ❌                            | ❌                              | ❌                          | ❌                                                                             | ❌                           | ❌               |
+| Sequelize DataType                                                                 | PostgreSQL                                                      | MariaDB                            | MySQL                        | MSSQL                          | [SQLite][sqlite-datatypes] | [Snowflake](https://docs.snowflake.com/en/sql-reference/data-types-text.html) | db2                         | ibmi            | Oracle           |
```
Comment (ephys) [inline docs/other-topics/other-data-types.mdx:30]: Can you add links to the Oracle DataTypes documentation like for the other dialects?

File: docs/other-topics/dialect-specific-things.md [hunk 2]
```diff
@@ -262,6 +262,36 @@ const sequelize = new Sequelize('MY_DSN', 'username', 'password', {
...
+
+Sequelize also lets you pass credentials in URL format:
+
+```js
+const sequelize = new Sequelize('oracle://user:pass@hostname:port/servicename');
+```
+
+:::note
+
+The `Sequelize.Datatypes.TEXT` doesn't support the `where` option.
```
Comment (ephys) [inline docs/other-topics/dialect-specific-things.md:292]:
```suggestion
`DataTypes.TEXT` doesn't support the `where` option. 
```

I think this belongs on the other-data-types page instead

File: docs/other-topics/dialect-specific-things.md [hunk 3]
```diff
@@ -262,6 +262,36 @@ const sequelize = new Sequelize('MY_DSN', 'username', 'password', {
...
+
+node-oracledb needs [instant client library files](https://www.oracle.com/database/technologies/instant-client/downloads.html) to work. You can use node-oracledb [quick start](https://oracle.github.io/node-oracledb/INSTALL.html#quickstart) link for installations.
+
+Below is Sequelize constructor with parameters related to Oracle Database.
+
+```js
+const sequelize = new Sequelize('servicename', 'username', 'password', {
+  dialect: 'oracle',
+  host: 'hostname',
+  [port: 'port number',]
```
Comment (ephys) [inline docs/other-topics/dialect-specific-things.md:278]: To keep this valid JS :)

```suggestion
  port: 'port number', // optional
```

However all of these options are standard. I saw a few dialectOptions on the implementation PR, can you document them and add a few of them as examples in this snippet? (or point to where they are documented in the underlying library)

**Review by ephys — CHANGES_REQUESTED** [review]
This PR modifies the documentation for v7 instead of v6, but [the implementation pr](https://github.com/sequelize/sequelize/pull/14638) is targeting v6

The v6 docs is in `versioned_docs`

I would recommend opening a new PR for the v6 docs as this PR can be kept open until the v7 implementation is done :)

**Review by cjbj — CHANGES_REQUESTED** [review]
Don't forget to add Oracle to the tagline in docusaurus.config.js :)

File: src/pages/releases.mdx [hunk 4]
```diff
@@ -161,6 +161,19 @@ support for Db2 for IBM i is limited as it is not handled by the core team.
...
 [odbc]: https://www.npmjs.com/package/odbc
 
+## Oracle Database Support Table
+
+Oracle Database requires the use of the [node-oracledb] npm package.  
+[Read more about this here](/docs/v7/other-topics/dialect-specific-things/#oracle).
+
+| Sequelize   | [Oracle Database]   | [node-oracledb] |
+|-------------|---------------------|-----------------|
+| 7 (alpha)   |      unknown        |    unknown      |
```
[the remark this answers is not in the document]
Comment (sudarshan12s) [reply src/pages/releases.mdx]: updated to upcoming.

**Comment by sudarshan12s** [comment]
> This PR modifies the documentation for v7 instead of v6, but [the implementation pr](https://github.com/sequelize/sequelize/pull/14638) is targeting v6
> 
> The v6 docs is in `versioned_docs`
> 
> I would recommend opening a new PR for the v6 docs as this PR can be kept open until the v7 implementation is done :)

A separate PR for v6 has been opened.
https://github.com/sequelize/website/pull/208

File: src/pages/releases.mdx (hunk 4 above)
[the remark this answers is not in the document]
Comment (sudarshan12s) [reply src/pages/releases.mdx]: Done.

File: docs/other-topics/dialect-specific-things.md (hunk 2 above)
Comment (sudarshan12s) [reply docs/other-topics/dialect-specific-things.md]: A note is added in other-data-types

**Comment by sudarshan12s** [comment]
> This PR modifies the documentation for v7 instead of v6, but [the implementation pr](https://github.com/sequelize/sequelize/pull/14638) is targeting v6
> 
> The v6 docs is in `versioned_docs`
> 
> I would recommend opening a new PR for the v6 docs as this PR can be kept open until the v7 implementation is done :)
new PR for v6 is captured here. 
https://github.com/sequelize/website/pull/210

File: docs/other-topics/dialect-specific-things.md (hunk 3 above)
Comment (sudarshan12s) [reply docs/other-topics/dialect-specific-things.md]: We have not added dialect specific parameters in the PR for now . Yeah We might add it later.

**Comment by sudarshan12s** [comment]
closing it for now.  v6 doc [PR](https://github.com/sequelize/sequelize/pull/14638)  has been merged. I shall re-open this when v7 code changes for Oracle dialect are ready.

**Pull request closed without merging**